symplyphysics/symbols/electrodynamics.py

```diff
@@ -0,0 +1,142 @@
+"""
+
+charge = SymbolNew("q", units.charge)
+"""
+**Electric charge** is the physical property of matter that causes it to experience a force when placed in an electromagnetic field.
+It can be positive or negative. Like charges repel each other and unlike charges attract each other.
+"""
+
+voltage = SymbolNew("V", units.voltage)
+"""
+**Voltage** is the difference in electric potential between two points.
+"""
+
+current = SymbolNew("I", units.current)
+"""
+**Current** is a flow of charged particles moving through an electrical conductor or space.
+"""
+
+resistance = SymbolNew("R", units.impedance)
+"""
+**Resistance** is the measure of the degree to which a conductor opposes an electric current through that conductor.
+It is the real part of the complex-valued impedance.
+"""
+
+electric_dipole_moment = SymbolNew("p", units.charge * units.length)
+"""
+**Electric dipole moment** is a measure of the separation of positive and negative electrical charges within a system:
+that is, a measure of the system's overall polarity.
+"""
+
+electric_field_strength = SymbolNew("E", units.voltage / units.length)
+"""
+**Electric field strength** refers to the magnitude of the electric field.
+"""
+
+surface_charge_density = SymbolNew("sigma", units.charge / units.area, display_latex="\\sigma")
+"""
+**Surface charge density** is charge per unit surface area.
+"""
+
+electric_flux = SymbolNew("Phi_E", units.voltage * units.length, display_latex="\\Phi_\mathbf{E}")
+"""
+**Electric flux** through a surface is the surface integral of the normal component of the electric field :math:`\mathbf{E}`
+over that surface.
+"""
+
+magnetic_flux_density = SymbolNew("B", units.magnetic_flux_density)
+"""
+**Magnetic flux density**, also called **magnetic induction**, is a physical quantity that predicts the force on a charged
+particle in the Lorentz force law.
+"""
+
+electric_potential = SymbolNew("U_E", units.voltage, display_latex="U_\mathbf{E}")
+"""
+**Electric potential** is defined as the amount of work or energy needed per unit of electric charge to move the charge from
+a reference point to a specific point in an electric field.
+"""
+
+power_factor = SymbolNew("pf", dimensionless, display_latex="\\mathrm{pf}")
+"""
+**Power factor** of an AC power system is defined as the ratio of the real power absorbed by the load to the apparent power
+flowing in the circuit.
+"""
+
+resistivity = SymbolNew("rho", units.impedance * units.length, display_latex="\\rho")
+"""
+**Electrical resistivity** is a fundamental specific property of a material that measures its electrical resistance or how
+strongly it resists electric current.
+"""
+
+inductance = SymbolNew("L", units.inductance)
+"""
+**Inductance** is the tendency of an electrical conductor to oppose a change in the electric current flowing through it.
+"""
+
+time_constant = SymbolNew("tau", units.time, display_latex="\\tau")
```

Comment:
```suggestion
electric_time_constant = SymbolNew("tau", units.time, display_latex="\\tau")
```

Reply:
I haven't found any mentions of time constants in other domains of physics though.

Reply:
Yes, but it sounds too general this way.

Reply:
Okay